test/com/facebook/buck/artifact_cache/DirArtifactCacheTest.java

```diff
@@ -300,9 +300,12 @@ public void testCacheStoresAndFetchHits() throws IOException {
         .map(input -> input.getFileName().toString())
         .collect(MoreCollectors.toImmutableSet());
 
+    /* Remove comparisson that not used, as it triggers CheckReturnValue
+     * (see http://errorprone.info/bugpattern/CheckReturnValue)
+     */
     for (RuleKey ruleKey : ImmutableSet.of(ruleKeyX, ruleKeyY, ruleKeyZ)) {
-      filenames.contains(ruleKey.toString());
-      filenames.contains(ruleKey.toString() + ".metadata");
+      assertTrue(filenames.contains(ruleKey.toString()));
```

Comment:
assertThat(filenames, Matchers.contains(

would give a nicer error message, but what you've got here is definitely an improvement on the status quo so I'd also accept it as-is if you can't be bothered :)